src/components v2/Atoms/Forms/InputText/InputText.vue

```diff
@@ -68,26 +65,32 @@ export default Vue.extend({
       default: '',
     },
     value: {
-      type: [Number, String] as PropType<InputValueType>,
+      type: [Number, String] as PropType<InputValues>,
       default: null,
     },
+    id: {
+      type: String as PropType<string>,
+      default: '',
+    },
+    type: {
+      type: String as PropType<InputTypes>,
+      default: 'text',
+    },
+    disabled: {
+      type: Boolean as PropType<boolean>,
+      default: false,
+    },
   },
 
   data(): InputTextData {
     return {
       inFocus: false,
-      currencyOptions: {
-        currency: null,
-        precision: 0,
-        valueRange: { min: 0 },
-        allowNegative: false,
-      },
-      id: '',
+      id_uid: '',
```

Comment:
No need to have this here as a prop. we can pass the ID straight through. That moves the `mounted` call away.